Gir.toml

```diff
@@ -383,25 +354,6 @@ generate_builder = true
     name = "activate-link"
     inhibit = true
 
-[[object]]
-name = "Gtk.AccelGroup"
```

Comment:
You also need to `git rm src/auto/accel_group.rs` etc for these. Did you do that for the other g*k4 crates? :)

Reply:
Yes, the make regen_check test takes care of ensuring that (it's run in the CI now)